### PR TITLE
Add in diffs between 2.3 and 2.2.2

### PR DESCRIPTION
**Pull request by kestewart**
Add in summary of key changes to the 2.3 version of the specification, compared to prior versions.

Signed-off-by: Kate Stewart <kate.stewart@att.net>

### CODE REVIEW

File: chapters/diffs-from-previous-editions.md
Comment (goneall) [inline chapters/diffs-from-previous-editions.md:10]: Propose changing:
rather than mandatory and requiring a “NOASSERTION” value.
to:
rather than requiring the use of "NOASSERTION" when no value is provided.

File: chapters/diffs-from-previous-editions.md
Comment (goneall) [inline chapters/diffs-from-previous-editions.md:13]: Nit - noticed an inconsistency in spaces after/before parenthesis

**Review by goneall — CHANGES_REQUESTED** [review]
Just a couple suggestions

File: chapters/diffs-from-previous-editions.md
Comment (goneall) [inline chapters/diffs-from-previous-editions.md:16]: Add the new relationship types:
REQUIREMENT_DESCRIPTION_FOR
SPECIFICATION_FOR

File: chapters/diffs-from-previous-editions.md
Comment (kestewart) [reply chapters/diffs-from-previous-editions.md:10]: That's a better way to phrase it.  Applied.   Thanks!

File: chapters/diffs-from-previous-editions.md
Comment (kestewart) [reply chapters/diffs-from-previous-editions.md:13]: FIxed.

**Comment by seabass-labrax** [comment]
I'll review this before tomorrow.

File: chapters/diffs-from-previous-editions.md
Comment (kestewart) [reply chapters/diffs-from-previous-editions.md:18]: Good catch.  Added.

**Comment by kestewart** [comment]
@goneall - I've made the updates you recommended, and a couple more.   Can you re-review please.

**Review by goneall — APPROVED** [review]
LGTM

**Comment by goneall** [comment]
I'll wait until @seabass-labrax review before merging tomorrow.

**Comment by kestewart** [comment]
@goneall - I think @seabass-labrax has run out of cycles.   Can you merge please and tag V2.3-rc1.   Would like to get the 2 week window started.

**Comment by seabass-labrax** [comment]
Sorry - yes, I've left this slip :/
Please merge this anyway and I'll review the rc as soon as possible.

**Review by zvr — CHANGES_REQUESTED** [review]
The one uses "Added", and "Update", which is inconsistent.

Previous change diffs (see below) use "has been added" and "has been updared"

**Comment by goneall** [comment]
@kestewart Can you add a line in to reflect PR #761 when you make the change requested by @zvr above?

Here's a proposed line:

"Update license matching guidelines Annex B to use the License List XML format"

@jlovejoy Has one change request for PR #761 , once that's made I plan to merge it in along with this PR and send out the email

**Comment by goneall** [comment]
I'll go ahead and merge this and create a separate PR with the change recommended above.

**Pull request merged**
